prometheus_client/metrics.py

```diff
@@ -579,7 +579,7 @@ def _metric_init(self):
                 self._labelvalues + (floatToGoString(b),))
             )
 
-    def observe(self, amount, exemplar=None):
+    def observe(self, amount: float = 1, exemplar: Optional[Dict[str, str]] = None) -> None:
```

Comment:
I believe the amount should not have a default value here.

Reply:
Ooops, sorry

Reply:
Fixed